### DISCUSSION TITLE
Is the list of TC39 proposals always up-to-date?

### DISCUSSION THREAD

**Discussion by nyngwang** [babel/babel#16338] [Q&A] [answered]
Say I want to know the latest transformations for experimental features, is [the list](https://babeljs.io/docs/plugins-list#tc39-proposals) in the website of Babel the upstream of this information?

Thanks!

**Comment by liuxingbaoyu** [comment, marked as answer]
No, this is not necessarily the latest.

**Reply by nyngwang** [reply]
Then, out of curiosity, would Babel provide a plugin for every latest feature? If so what's the delay (roughly) between the proposal coming out and the proposal implemented as a Babel plugin?

**background**: since now I'm using Babel it will be nice to know how far I can go using it.

**Reply by liuxingbaoyu** [reply]
No, generally only proposals that reach the stage 3 will be implemented. This is usually completed within a few months.
